erpnext/accounts/doctype/sales_invoice/sales_invoice.py

```diff
@@ -352,12 +344,6 @@ def on_cancel(self):
 
 		unlink_inter_company_doc(self.doctype, self.name, self.inter_company_invoice_reference)
 
-		# Healthcare Service Invoice.
-		domain_settings = frappe.get_doc('Domain Settings')
-		active_domains = [d.domain for d in domain_settings.active_domains]
-
-		if "Healthcare" in active_domains:
-			manage_invoice_submit_cancel(self, "on_cancel")
```

Comment:
There is a `set_healthcare_services` method in this file. Even that needs to be removed.

https://github.com/frappe/erpnext/blob/c8565c7090d153cba83d4c08423dafa941582e10/erpnext/accounts/doctype/sales_invoice/sales_invoice.py#L1421